lib/node_modules/@stdlib/math/base/special/hyp2f1/test/test.js

```diff
@@ -0,0 +1,295 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var isnanf = require( '@stdlib/math/base/assert/is-nanf' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var hyp2f1 = require( './../lib' );
+
+
+// FIXTURES //
+
+var basic = require( './fixtures/python/basic.json' );
+var edgeCases1 = require( './fixtures/python/edge_cases1.json' );
+var edgeCases2 = require( './fixtures/python/edge_cases2.json' );
+var edgeCases3 = require( './fixtures/python/edge_cases3.json' );
+var edgeCases4 = require( './fixtures/python/edge_cases4.json' );
+var outliers = require( './fixtures/python/outliers.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof hyp2f1, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `1` if `x` is 0', function test( t ) {
+	var v;
+
+	v = hyp2f1( 1.0, 1.0, 1.0, 0.0 );
+	t.strictEqual( v, 1.0, 'returns expected value' );
+
+	v = hyp2f1( 1.5, 2.5, 3.5, 0.0 );
+	t.strictEqual( v, 1.0, 'returns expected value' );
+
+	v = hyp2f1( -1, 4, 2, 0.0 );
+	t.strictEqual( v, 1.0, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `NaN`', function test( t ) {
+	var v;
+
+	v = hyp2f1( NaN, 3.0, 2.0, 0.5 );
+	t.equal( isnanf( v ), true, 'returns expected value' );
+
+	v = hyp2f1( 0.0, NaN, 2.0, 0.5 );
+	t.equal( isnanf( v ), true, 'returns expected value' );
+
+	v = hyp2f1( 0.0, 3.0, NaN, 0.5 );
+	t.equal( isnanf( v ), true, 'returns expected value' );
+
+	v = hyp2f1( 0.0, 3.0, 2.0, NaN );
+	t.equal( isnanf( v ), true, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'returns `1` if either `a` or `b` is 0 and `c` is not 0', function test( t ) {
```

Comment:
```suggestion
tape( 'the function returns `1` if either `a` or `b` is 0 and `c` is not 0', function test( t ) {
```

Applies here and elsewhere.